frontend/src/utils/interfaces.ts

```diff
@@ -0,0 +1,29 @@
+// User interface
+interface IAuthoritiesObj {
+  authority: string
+};
+
+interface IRoleObj {
+  roleId?: number,

```

Comment:
is the possible that roleId can be empty?

Reply:
Added 'roleId' in the UserState separately (not as a part of 'role' object)